nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-1-0-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/ConsumeKafkaRecord_1_0.java

```diff
@@ -86,12 +87,15 @@
     static final AllowableValue TOPIC_NAME = new AllowableValue("names", "names", "Topic is a full topic name or comma separated list of names");
     static final AllowableValue TOPIC_PATTERN = new AllowableValue("pattern", "pattern", "Topic is a regex using the Java Pattern syntax");
 
+    static final int TOPIC_LIST_MAX_LENGTH = 1000;
```

Comment:
I agree with @joewitt that we cannot go up 10x without a serious performance evaluation.